[0.058s] Invoking command in '/home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/build/Interfaces_to_gazebo': CMAKE_PREFIX_PATH=/home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/install/ros2_gazebo_estudo:/opt/ros/humble /usr/bin/cmake /home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/Interfaces_to_gazebo -DCMAKE_INSTALL_PREFIX=/home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/install/Interfaces_to_gazebo
[1.378s] -- The C compiler identification is GNU 11.3.0
[2.380s] -- The CXX compiler identification is GNU 11.3.0
[2.495s] -- Detecting C compiler ABI info
[2.794s] -- Detecting C compiler ABI info - done
[2.815s] -- Check for working C compiler: /usr/bin/cc - skipped
[2.816s] -- Detecting C compile features
[2.841s] -- Detecting C compile features - done
[2.862s] -- Detecting CXX compiler ABI info
[3.092s] -- Detecting CXX compiler ABI info - done
[3.114s] -- Check for working CXX compiler: /usr/bin/c++ - skipped
[3.117s] -- Detecting CXX compile features
[3.118s] -- Detecting CXX compile features - done
[3.239s] -- Found ament_cmake: 1.3.3 (/opt/ros/humble/share/ament_cmake/cmake)
[3.822s] -- Found Python3: /usr/bin/python3.10 (found version "3.10.6") found components: Interpreter 
[4.225s] -- Found ament_lint_auto: 0.12.5 (/opt/ros/humble/share/ament_lint_auto/cmake)
[4.379s] WARNING: Package name "Interfaces_to_gazebo" does not follow the naming conventions. It should start with a lower case letter and only contain lower case letters, digits, underscores, and dashes.
[4.513s] -- Found geometry_msgs: 4.2.3 (/opt/ros/humble/share/geometry_msgs/cmake)
[4.974s] -- Found rosidl_generator_c: 3.1.4 (/opt/ros/humble/share/rosidl_generator_c/cmake)
[5.015s] -- Found rosidl_adapter: 3.1.4 (/opt/ros/humble/share/rosidl_adapter/cmake)
[5.098s] -- Found rosidl_generator_cpp: 3.1.4 (/opt/ros/humble/share/rosidl_generator_cpp/cmake)
[5.545s] -- Using all available rosidl_typesupport_c: rosidl_typesupport_fastrtps_c;rosidl_typesupport_introspection_c
[5.654s] -- Using all available rosidl_typesupport_cpp: rosidl_typesupport_fastrtps_cpp;rosidl_typesupport_introspection_cpp
[6.201s] -- Found rosidl_default_generators: 1.2.0 (/opt/ros/humble/share/rosidl_default_generators/cmake)
[7.021s] [31mCMake Error at /opt/ros/humble/share/rosidl_adapter/cmake/rosidl_adapt_interfaces.cmake:59 (message):
[7.021s]   execute_process(/usr/bin/python3.10 -m rosidl_adapter --package-name
[7.022s]   Interfaces_to_gazebo --arguments-file
[7.022s]   /home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/build/Interfaces_to_gazebo/rosidl_adapter__arguments__Interfaces_to_gazebo.json
[7.022s]   --output-dir
[7.023s]   /home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/build/Interfaces_to_gazebo/rosidl_adapter/Interfaces_to_gazebo
[7.023s]   --output-file
[7.023s]   /home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/build/Interfaces_to_gazebo/rosidl_adapter/Interfaces_to_gazebo.idls)
[7.024s]   returned error code 1:
[7.024s] 
[7.024s]   Traceback (most recent call last):
[7.025s] 
[7.025s]     File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
[7.025s]       return _run_code(code, main_globals, None,
[7.025s]     File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
[7.026s]       exec(code, run_globals)
[7.026s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__main__.py", line 19, in <module>
[7.026s]       sys.exit(main())
[7.026s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/main.py", line 53, in main
[7.027s]       abs_idl_file = convert_to_idl(
[7.027s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/__init__.py", line 24, in convert_to_idl
[7.027s]       return convert_srv_to_idl(
[7.027s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/srv/__init__.py", line 28, in convert_srv_to_idl
[7.028s]       srv = parse_service_string(package_name, input_file.stem, content)
[7.028s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 859, in parse_service_string
[7.028s]       request_message = parse_message_string(
[7.029s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 544, in parse_message_string
[7.040s]       msg = MessageSpecification(pkg_name, msg_name, fields, constants)
[7.040s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 388, in __init__
[7.041s]       self.base_type = BaseType(
[7.042s]     File "/opt/ros/humble/local/lib/python3.10/dist-packages/rosidl_adapter/parser.py", line 201, in __init__
[7.043s]       raise InvalidResourceName(
[7.049s] 
[7.049s]   rosidl_adapter.parser.InvalidResourceName: 'Interfaces_to_gazebo' is an
[7.050s]   invalid package name.  It should have the pattern
[7.050s]   '^(?!.*__)(?!.*_$)[a-z][a-z0-9_]*$'
[7.050s] 
[7.050s] Call Stack (most recent call first):
[7.051s]   /opt/ros/humble/share/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake:130 (rosidl_adapt_interfaces)
[7.051s]   CMakeLists.txt:29 (rosidl_generate_interfaces)
[7.051s] 
[7.052s] [0m
[7.052s] -- Configuring incomplete, errors occurred!
[7.053s] See also "/home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/build/Interfaces_to_gazebo/CMakeFiles/CMakeOutput.log".
[7.053s] Invoked command in '/home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/build/Interfaces_to_gazebo' returned '1': CMAKE_PREFIX_PATH=/home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/install/ros2_gazebo_estudo:/opt/ros/humble /usr/bin/cmake /home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/Interfaces_to_gazebo -DCMAKE_INSTALL_PREFIX=/home/andre/Desktop/Git/ROS_estudo/Simulacao_ROS2/install/Interfaces_to_gazebo
